react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,39 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
+  const { id } = orderForm
+
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
```

Comment:
```suggestion
    function getCustomerClass() {
      const requestOptions = {
        method: 'GET',
      }
      
      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
    }
```

Reply:
@igorbrasileiro Updated the code using async/await

Reply:
@igorbrasileiro Changed the code as per suggestion